src/Stratis.Bitcoin.Features.Miner/PowMining.cs

```diff
@@ -125,7 +139,7 @@ public List<uint256> GenerateBlocks(ReserveScript reserveScript, ulong generate,
 
                 if (Block.BlockSignature)
                 {
-                    // POS: make sure the POS consensus rules are valid 
+                    // POS: make sure the POS consensus rules are valid
```

Comment:
missing dot and we can even make it better comment, maybe something like 

`Make sure the POS consensus rules are valid. This is required for generation of blocks inside tests, where it is possible to generate multiple blocks within one second.`